### DISCUSSION TITLE
Custom CSS

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#33] [Ideas]
Some firefox addons have their own editor box where you can paste your own CSS which overrides the CSS of the addon.
One of the addons is quick tabs ported:
![custom css](https://github.com/madprops/grasshopper/assets/74585843/b30d0ea9-224d-4f58-9e2f-f6f9bd7369e4)

If would love such a feature in the near future if you don't mind.

**Comment by madprops** [comment]
Yeah this seems easy enough to do.

**Comment by madprops** [comment]
Done.

**Comment by N3C2L** [comment]
Thanks! 
I know that you can resize the editor, but maybe not every user knows that. Hence I would resize the default editor size to 4x the size at it is now. Another thing: I would set the text alignment in the editor to left-justified.

**Comment by madprops** [comment]
It's not an editor, it's only an area to paste text. You're not supposed to edit the css there.
If it was an editor i'd use the Ace editor, but it's a big library and gets messy.
I might add an Edit option in the setting titles of text items to open the "editor" I already have though.

**Comment by N3C2L** [comment]
I am sorry if the word "editor" means for me any place where text can be edited. Please forgive me, because from a non-developer view this would have been correct. But I can understand that developers these days would call an "editor" then an editor if it has syntax highlighting etc.

**Comment by madprops** [comment]
Next version you will be able to Edit more fully by clicking `Custom CSS` title, there's an Edit option, i'll upload soon.

**Comment by madprops** [comment]
Done. 

![edit345](https://github.com/madprops/grasshopper/assets/206498/8391e6f1-8726-4f25-8bd3-3cdf665d9c9f)

And you're right in that a textarea is an editor, just not really meant to edit CSS since it's uncomfortable compared to a text editor in your computer.

Stuff like this do turn it into a good editor though: https://github.com/ajaxorg/ace

**Comment by N3C2L** [comment]
Alright, thanks.
